framework/src/modules/interoperability/schema.ts

```diff
@@ -468,3 +476,79 @@ export const sidechainTerminatedCCMParamsSchema = {
 		},
 	},
 };
+
+export const nameSchema = {
+	$id: 'modules/interoperability/name',
+	type: 'object',
+	required: ['name'],
+	properties: {
+		name: {
+			dataType: 'string',
+			fieldNumber: 1,
+		},
+	},
+};
+
+// Note: Changed to lower-case `id` as ajv requires it
+export const chainIDSchema = {
+	$id: 'modules/interoperability/chainId',
+	type: 'object',
+	required: ['id'],
+	properties: {
+		id: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+	},
+};
+
+export const validatorsSchema = {
+	$id: 'modules/interoperability/validators',
+	type: 'object',
+	required: ['activeValidators', 'certificateThreshold'],
+	properties: {
+		activeValidators: {
+			type: 'array',
+			fieldNumber: 1,
+			items: {
+				type: 'object',
+				required: ['blsKey', 'bftWeight'],
+				properties: {
+					blsKey: {
+						dataType: 'bytes',
+						fieldNumber: 1,
```

Comment:
I think you could also include the minLength, maxLength here. Similar to [#7147(comment)](https://github.com/LiskHQ/lisk-sdk/pull/7147#discussion_r865306622)